### PR TITLE
Fixes #48 : changed localhost to .env url

### LINKED ISSUE #48: Making Nachet-Frontend ready for deployment
The client wants a working version of Nachet to be deployed so they can test it out. Yesterday, as we were looking into deploying the front-end, we must manually change all mentions of http://localhost:8080 to the appropriate Google Cloud Platform URL in the src/root/body/index.tsx file. 

Example: 
image

Ricky said you had already done something similar in Louis. We would want the file to be generic and work in any deployment context.

### PR DESCRIPTION
**Pull request by redat97**
Added .env.template file. 
Because it uses CRA I added a function that is called getBackendUrl() in index.tsx file

### CODE REVIEW

File: src/root/body/index.tsx [hunk 1]
```diff
@@ -117,6 +117,14 @@ const Body: React.FC<params> = (props) => {
     }
   };
 
+  const getBackendUrl = (): string => {
+    const backendURL = process.env.REACT_APP_BACKEND_URL;
+    if (backendURL === null || backendURL === undefined || backendURL === "") {
+      throw new Error("REACT_APP_BACKEND_URL·is·not·set·in·.env");
```
Comment (rngadam) [inline src/root/body/index.tsx:123]: it`s not necessarily .env.... it's an environment variable that gets loaded from either .env or ... set in the environment. don't refer it as .env

File: src/root/body/index.tsx [hunk 2]
```diff
@@ -117,6 +117,14 @@ const Body: React.FC<params> = (props) => {
     }
   };
 
+  const getBackendUrl = (): string => {
```
Comment (rngadam) [inline src/root/body/index.tsx:120]: not sure we want to wait upon a call to the backend to fail; we would want to detect if the backend is configured and working (with a new /ping endpoint) as soon as the frontend comes up. We can postpone that in a future issue and have @CFIALeronB  work on it.

**Review by rngadam — CHANGES_REQUESTED** [review]

File: .env.template [hunk 3]
```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL=
```
Comment (CFIALeronB) [inline .env.template:1]: Space needs to be added to EOF

File: src/root/body/index.tsx (hunk 2 above)
Comment (RussellJimmies) [reply src/root/body/index.tsx:120]: https://github.com/ai-cfia/nachet-frontend/issues/55

**Pull request merged**